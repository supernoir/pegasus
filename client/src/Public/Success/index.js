import React from 'react'
import {CssBaseline, Box, Paper, Typography} from '@material-ui/core'
import {Menu} from '../../base/Menu'
import {Footer} from '../../base/Footer'
import {Header} from '../components/Header/index'
import Scale from '../components/Scale/index'

export default class Success extends React.Component {
	render() {
		return (
			<React.Fragment>
				<CssBaseline />
				<Menu />
				<Box p={4}>
					<Paper>
						<Box p={3}>
							<Typography component="h2" variant="h2" align="center" color="textPrimary">
								Thank you for your time
							</Typography>
						</Box>
					</Paper>
				</Box>
			</React.Fragment>
		)
	}
}
