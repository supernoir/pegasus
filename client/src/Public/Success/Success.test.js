import React from 'react'
import renderer from 'react-test-renderer'
import Success from './index'

test('Component -- Success', () => {
	const component = renderer.create(<Success />)
	let tree = component.toJSON()
	expect(tree).toMatchSnapshot()
})
