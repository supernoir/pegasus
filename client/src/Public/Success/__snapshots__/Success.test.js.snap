// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component -- Success 1`] = `
Array [
  <header
    className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionRelative MuiAppBar-colorPrimary"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h6
        className="MuiTypography-root MuiTypography-h6 MuiTypography-colorInherit MuiTypography-noWrap"
      >
        PegaSUS
      </h6>
      <nav>
        <a
          aria-current="page"
          aria-disabled={false}
          className="MuiButtonBase-root MuiButton-root MuiButton-text"
          href="/"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="MuiButton-label"
          >
            Survey
          </span>
        </a>
        <a
          aria-disabled={false}
          className="MuiButtonBase-root MuiButton-root MuiButton-text"
          href="/dashboard"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="MuiButton-label"
          >
            Dashboard
          </span>
        </a>
        <a
          aria-disabled={false}
          className="MuiButtonBase-root MuiButton-root MuiButton-text"
          href="/admin/"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="MuiButton-label"
          >
            Administration
          </span>
        </a>
      </nav>
    </div>
  </header>,
  <div
    className="MuiBox-root MuiBox-root-103"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
    >
      <div
        className="MuiBox-root MuiBox-root-104"
      >
        <h2
          className="MuiTypography-root MuiTypography-h2 MuiTypography-colorTextPrimary MuiTypography-alignCenter"
        >
          Thank you for your time
        </h2>
      </div>
    </div>
  </div>,
]
`;
